geoportal/demo_geoportal/locale/de/LC_MESSAGES/demo_geoportal-client.po

```diff
@@ -257,7 +259,7 @@ msgstr "Marke"
 
 #, fuzzy
 msgid "buffer_stop"
-msgstr "Bushaltestelle"
+msgstr ""
```

Comment:
Wanted?

Reply:
Yes. I don't know what "buffer_stop", but it certainly isn't a "Bushaltestelle" (bus stop)

Reply:
Effectively: https://www.google.ch/search?client=ubuntu&hs=lyq&channel=fs&dcr=0&biw=1469&bih=972&tbm=isch&sa=1&ei=xgm5Wt2iMMOyswGO3piYAg&q=buffer+stop&oq=buffer+stop&gs_l=psy-ab.3..0i19k1l10.1328.3900.0.4260.3.3.0.0.0.0.128.338.0j3.3.0....0...1c..64.psy-ab..0.3.338....0.vqnH-KW-FZ0 :-)

Reply:
Ah, so that's a buffer stop... OK, I will add a translation.